backend/app/services/device/local_provider.py

```
@@ -0,0 +1,505 @@
+        include_offline: bool = True,
+    ) -> List[Dict[str, Any]]:
+        """List all local devices for a user."""
+        # Get all Device CRDs for user
+        devices = (
+            db.query(Kind)
+            .filter(
+                and_(
+                    Kind.user_id == user_id,
+                    Kind.kind == "Device",
+                    Kind.namespace == "default",
+                    Kind.is_active == True,
```

Ruff (0.15.0)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)